third_party/3/waitress/__init__.pyi

```diff
@@ -0,0 +1,6 @@
+from typing import Any, Tuple
+from waitress.server import create_server
+
+def serve(app: Any, **kw: Any) -> None: ...
+def serve_paste(app: Any, global_conf: None, **kw: str) -> int: ...
```

Comment:
`global_conf` seems to be ignored. Wouldn't it be better to type it as `Any`? Also, `kw` -> `Any`.